casepro/cases/models.py

```diff
@@ -332,11 +332,12 @@ def reopen(self, user, note=None, update_contact=True):
         self.notify_watchers(action=action)
 
     @case_action()
-    def reassign(self, user, partner, note=None):
+    def reassign(self, user, partner, user_assignee=None, note=None):
```

Comment:
Wondering whether `user_assignee` should be the last param to maintain some backwards compatibility, though if tests are passing maybe this isn't a big deal.